src/r/recordFactories/ProjectFactory.ts

```diff
@@ -701,15 +799,12 @@ export class ProjectFactory extends RecordFactory<RT.project> {
    * @param obj
    * @param position
    */
-  pasteFromClipboardObject(this: ProjectFactory, obj: ClipboardR, position?: number): void {
-    if(obj.parentType !== this._type) {
-      console.error(`Can't paste this object into a RecordNode of type of ${this._type}`);
-      return;
-    }
+  pasteFromClipboardObject(this: ProjectFactory, { obj, position, groupElementId, sceneId }:  {obj: ClipboardR, position?: number, groupElementId?: number, sceneId?: number}): void {
     const projectVars = this.getRecords(RT.variable);
     const scenesFromClipboard = obj.nodes.filter(s => s.type === RT.scene);
     const variablesFromClipboard = obj.nodes.filter(s => s.type === RT.variable);
     const otherRecordsFromClipboard = obj.nodes.filter(s => s.type !== RT.variable && s.type !== RT.scene);
```

Comment:
Fix the condition here, exclude elements from this list

Reply:
Add test cases around this. This wasn't tested properly

Reply:
This still hasn't been fixed and no test cases added here. 

Test case: 
Clipboard objects contains elements and scenes. 
Try pasting a clipboard object of this format, the compiler will throw warnings. Elements can't be added at project level